packages/hardhat-plugin/src/index.ts

```diff
@@ -15,6 +15,7 @@ import {
 } from "fs-extra";
 import { extendConfig, extendEnvironment, scope } from "hardhat/config";
 import { NomicLabsHardhatPluginError } from "hardhat/plugins";
+import { parse as json5parse } from "json5";
```

Comment:
I had to import like this to make the linter happy, I hope that is okay

Reply:
Can you move this down into the deploy task function instead? It should go on line 119 above the `prompts` import like this: 

```typescript
const { default: chalk } = await import("chalk");
const { parse: json5Parse } = await import("json5");
const { default: Prompt } = await import("prompts");
```